lib/prompts/accountNamePrompt.ts

```diff
@@ -1,36 +1,52 @@
-// @ts-nocheck
-const { accountNameExistsInConfig } = require('@hubspot/local-dev-lib/config');
-const { promptUser } = require('./promptUtils');
-const { i18n } = require('../lang');
-const {
-  HUBSPOT_ACCOUNT_TYPES,
-} = require('@hubspot/local-dev-lib/constants/config');
+import { accountNameExistsInConfig } from '@hubspot/local-dev-lib/config';
+import { promptUser } from './promptUtils';
+import { i18n } from '../lang';
+import { HUBSPOT_ACCOUNT_TYPES } from '@hubspot/local-dev-lib/constants/config';
 
 const i18nKey = 'lib.prompts.accountNamePrompt';
 
-const getCliAccountNamePromptConfig = defaultName => ({
-  name: 'name',
-  message: i18n(`${i18nKey}.enterAccountName`),
-  default: defaultName,
-  validate(val) {
-    if (typeof val !== 'string') {
-      return i18n(`${i18nKey}.errors.invalidName`);
-    } else if (!val.length) {
-      return i18n(`${i18nKey}.errors.nameRequired`);
-    } else if (val.indexOf(' ') >= 0) {
-      return i18n(`${i18nKey}.errors.spacesInName`);
-    }
-    return accountNameExistsInConfig(val)
-      ? i18n(`${i18nKey}.errors.accountNameExists`, { name: val })
-      : true;
-  },
-});
+type AccountNamePromptConfig = {
+  name: string;
+  message: string;
+  default: string;
+  validate(val: string): string | boolean;
+};
+
+export function getCliAccountNamePromptConfig(
+  defaultName: string
+): AccountNamePromptConfig {
+  return {
+    name: 'name',
+    message: i18n(`${i18nKey}.enterAccountName`),
+    default: defaultName,
+    validate(val: string): string | boolean {
+      if (typeof val !== 'string') {
+        return i18n(`${i18nKey}.errors.invalidName`);
+      } else if (!val.length) {
+        return i18n(`${i18nKey}.errors.nameRequired`);
+      } else if (val.indexOf(' ') >= 0) {
+        return i18n(`${i18nKey}.errors.spacesInName`);
+      }
+      return accountNameExistsInConfig(val)
+        ? i18n(`${i18nKey}.errors.accountNameExists`, { name: val })
+        : true;
+    },
+  };
+}
 
-const cliAccountNamePrompt = defaultName => {
+export function cliAccountNamePrompt(
+  defaultName: string
+): AccountNamePromptConfig {
   return promptUser(getCliAccountNamePromptConfig(defaultName));
-};
+}
 
-const hubspotAccountNamePrompt = ({ accountType, currentPortalCount = 0 }) => {
+export function hubspotAccountNamePrompt({
+  accountType,
+  currentPortalCount = 0,
+}: {
+  accountType: keyof typeof HUBSPOT_ACCOUNT_TYPES;
```

Comment:
You can use the `AccountType` type from local-dev-lib

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1273/commits/dc551a7bade92e90b7e5f5994c7b96a0085f785a